_includes/tutorials/joining-stream-table/kstreams/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,19 @@
+Use the `builder.table()` method to create a `KTable`.
+Then use the `https://kafka.apache.org/{{ site.ak_javadoc_version }}/javadoc/org/apache/kafka/streams/kstream/ValueJoiner.html[ValueJoiner]` interface in the Streams API to join the `KStream` and `KTable`.
+
+```
+// Instantiate movie joiner class
```

Comment:
@awalther28 thank you for adding a short answer where there was not one before.

I think this can be simplified a lot though because right now it looks really busy and is hard to grok.  

I would suggest something like

```
KStream<String, Rating> ratings = ...
KTable<String, Movie> movies = ....
final MovieRatingJoiner joiner = new MovieRatingJoiner();
KStream<String, RatedMovie> ratedMovie = ratings.join(movies, joiner);
```

(Note: unlike other places, here the `...` literally means `...`)

Reply:
okay committed in https://github.com/confluentinc/kafka-tutorials/pull/827/commits/4619d631f56d6f894297fb2585ba93b3244647bf